oscarlab/graphene
# [LibOS] test/apps: Add Node.js Express server example

## Affected components


## Description of the changes 

This PR adds NodeJS Express Server example.

The corresponding PR is https://github.com/oscarlab/graphene-tests/pull/72. 

## How to test this PR? 


We already have NodeJS test. This PR doesn't add any tests, but manually one can check the new app via `make check` or `make SGX=1 check`.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1374)
